app/utils.py

```diff
@@ -54,16 +52,15 @@ def format_time(time: Union[int, float]) -> str:
     retry=retry_if_exception_type(asyncio.TimeoutError),
 )
 async def channel_message(channel: str, message: str) -> None:
-    async with ClientSession() as sesh:
-        await sesh.get(
-            "http://localhost:5001/api/v1/fokabotMessage",
-            params={
-                "to": channel,
-                "msg": message,
-                "k": config.FOKABOT_KEY,
-            },
-            timeout=2,
-        )
+    await app.state.services.http.get(
+        "http://localhost:5001/api/v1/fokabotMessage",
+        params={
+            "to": channel,
+            "msg": message,
+            "k": config.FOKABOT_KEY,
+        },
+        timeout=2,
+    )
```

Comment:
make this send_message_in_background, put strong retry policy, and spawn call with asyncio.create_task